src/guides/v2.3/install-gde/prereq/apache.md

```diff
@@ -37,6 +37,18 @@ Magento uses server rewrites and `.htaccess` to provide directory-level instruct
 {:.bs-callout-info}
 Failure to enable these settings typically results in no styles displaying on your storefront or Admin.
 
+## Apache required modules {#apache-required-modules}
+
+Magento requires the following Apache modules be installed:
+
+*  [mod_deflate.c](https://httpd.apache.org/docs/2.4/mod/mod_deflate.html)
+*  [mod_expires.c](https://httpd.apache.org/docs/2.4/mod/mod_expires.html)
+*  [mod_headers.c](https://httpd.apache.org/docs/2.4/mod/mod_headers.html)
+*  [mod_php7.c](https://www.php.net/manual/en/security.apache.php)
```

Comment:
The requirement of mod_php7 is not strictly true. If you are using php-fpm for example this is not necessary. I would remove this line, no?

Reply:
Hi, @hostep !

Good notice!

May be I should add an `(optional)` word for `mod_php7`.

What do you think? Thanks

Reply:
Maybe but then I would also change the `Magento requires ...` to `Magento recommends ...` or something like that?

Reply:
@atwixfirster Could you just remove it from the list? It's not actually required, as there few options how to set up apache configuration.

Reply:
> @atwixfirster Could you just remove it from the list? It's not actually required, as there few options how to set up apache configuration.

@ihor-sviziev ,

 **done**, sir